quarkus/extensions/pom.xml

```diff
@@ -32,6 +32,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
Is this needed now that the Provider is moved to services?

Reply:
done